website/pages/docs/k8s/crds/upgrade-to-crds.mdx

```diff
@@ -0,0 +1,318 @@
+---
+layout: docs
+page_title: Upgrade An Existing Cluster to CRDs
+sidebar_title: Upgrade An Existing Cluster to CRDs
+description: >-
+  Upgrade an existing cluster to use custom resources.
+---
+
+# Upgrade An Existing Cluster to CRDs
+
+-> This feature requires consul-helm >= 0.26.0 and consul >= 1.8.4.
+
+If you have an existing Consul cluster running on Kubernetes you may need to perform
+extra steps to migrate to CRDs.
+
+You will need to perform extra steps if you are using:
+
+- Helm config `connectInject.centralConfig.defaultProtocol`, e.g.
+
+  ```yaml
```

Comment:
This reads as a list, but because of the wording it's a little unclear.
Is there a way to bullet these sections? Or use `OR` or something between them?

Reply:
They're already bulleted, but I added `Or` to the sentences. Yeah it looks not great. Not sure how best to a) list the features that you'll need to migrate b) show example yaml.

Reply:
yeah its tricky, looks okay as is :)